LucidGloves/include/DeviceConfiguration.h

```diff
@@ -2,29 +2,36 @@
 #include "openvr_driver.h"
 #include "Comm/SerialCommunicationManager.h"
 
-enum VRDeviceProtocol {
+enum VRCommunicationProtocol {
 	SERIAL = 0,
 };
 
+enum VREncodingProtocol {
+	LEGACY = 0,
+};
+
 struct VRDeviceConfiguration_t {
-	VRDeviceConfiguration_t(vr::ETrackedControllerRole role, bool enabled, vr::HmdVector3_t offsetVector, vr::HmdVector3_t angleOffsetVector, float poseOffset, VRSerialConfiguration_t serialConfiguration) :
+	VRDeviceConfiguration_t(vr::ETrackedControllerRole role, bool enabled, vr::HmdVector3_t offsetVector, vr::HmdVector3_t angleOffsetVector, float poseOffset, int adcCounts, VRCommunicationProtocol communicationProtocol, VREncodingProtocol encodingProtocol, VRSerialConfiguration_t serialConfiguration) :
 		role(role),
 		enabled(enabled),
 		offsetVector(offsetVector),
 		angleOffsetVector(angleOffsetVector),
 		poseOffset(poseOffset),
+		adcCounts(adcCounts),
 		serialConfiguration(serialConfiguration),
-		protocol(VRDeviceProtocol::SERIAL) {};
+		communicationProtocol(communicationProtocol),
+		encodingProtocol(encodingProtocol) {};
 
 	vr::ETrackedControllerRole role;
 	bool enabled;
 	vr::HmdVector3_t offsetVector;
 	vr::HmdVector3_t angleOffsetVector;
-
+	int adcCounts;
```

Comment:
In `LegacyEncodingManager` it's called `maxAnalogValue`, is there any reason why they are called different names?

Reply:
Will fix for consistency